<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDisplay</class>
 <widget class="QWidget" name="frmDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmDisplay</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>371</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="SwitchButton" name="SwitchButtonTwo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>87</horstretch>
        <verstretch>28</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="SwitchButton" name="SwitchButtonOne">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>87</horstretch>
        <verstretch>28</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="SwitchButton" name="SwitchButtonThree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>87</horstretch>
        <verstretch>28</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QPushButton</extends>
   <header>switchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
